src/KM_Controls.pas

```diff
@@ -4610,7 +4628,7 @@ function TKMListBox.KeyDown(Key: Word; Shift: TShiftState): Boolean;
                 end;
     else        begin
                   if Assigned(OnKeyDown) then
-                    Result := OnKeyDown(Key, Shift);
+                    Result := OnKeyDown(Self, Key, Shift);

```

Comment:
Luckily it should not be invoked second time, because of
`if inherited KeyDown(Key, Shift) then Exit;`

But ofc its better to remove this. Same for TKMEdit